src/dashboard/server/api/router.js

```diff
@@ -25,6 +25,9 @@ router.get('/teams/:teamId/clusters/:clusterId',
 router.get('/clusters/:clusterId',
   require('./middlewares/user')(),
   require('./controllers/cluster'))
+router.get('/teams/:teamId/clusters/:clusterId/metadata',
+  require('./middlewares/user')(),
+  require('./controllers/team/metadata'))
```

Comment:
It's better to put the controller to
```suggestion
  require('./controllers/team/cluster/metadata'))
```